src/bun.zig

```diff
@@ -1836,7 +1836,7 @@ var needs_proc_self_workaround: bool = false;
 // necessary on linux because other platforms don't have an optional
 // /proc/self/fd
 fn getFdPathViaCWD(fd: std.os.fd_t, buf: *[@This().MAX_PATH_BYTES]u8) ![]u8 {
-    const prev_fd = try std.os.openatZ(std.fs.cwd().fd, ".", 0, 0);
+    const prev_fd = try std.os.openatZ(std.fs.cwd().fd, ".", std.os.O.DIRECTORY, 0);
```

Comment:
This will fail if the fd is for a file, but it would fail before this change too when fchdir is called

Reply:
`std.fs.cwd().fd` is never a file

but the fchdir call below - yeah. that fails